docs/ref/settings.txt

```diff
@@ -1141,11 +1141,17 @@ See also :setting:`DATETIME_FORMAT`, :setting:`TIME_FORMAT` and :setting:`SHORT_
 Default::
 
     [
-        '%Y-%m-%d', '%m/%d/%Y', '%m/%d/%y', # '2006-10-25', '10/25/2006', '10/25/06'
-        '%b %d %Y', '%b %d, %Y',            # 'Oct 25 2006', 'Oct 25, 2006'
-        '%d %b %Y', '%d %b, %Y',            # '25 Oct 2006', '25 Oct, 2006'
-        '%B %d %Y', '%B %d, %Y',            # 'October 25 2006', 'October 25, 2006'
-        '%d %B %Y', '%d %B, %Y',            # '25 October 2006', '25 October, 2006'
+        '%Y-%m-%d',  # '2006-10-25'
+        '%m/%d/%Y',  # '10/25/2006'
+        '%m/%d/%y',  # '10/25/06'
+        '%b %d %Y',  # 'Oct 25 2006'
+        '%b %d, %Y',  # 'Oct 25, 2006'
+        '%d %b %Y',  # '25 Oct 2006'
+        '%d %b, %Y',  # '25 Oct, 2006'
+        '%B %d %Y',  # 'October 25 2006'
+        '%B %d, %Y',  # 'October 25, 2006'
+        '%d %B %Y',  # '25 October 2006'
+        '%d %B, %Y',  # '25 October, 2006'
```

Comment:
So black will not update all of the code examples in the documentation.
I guess we could consider seeing if https://github.com/asottile/blacken-docs helps?

Reply:
I don't want to update any examples in docs, as far as I'm aware it's not described in DEP-8. I updated `DATE_INPUT_FORMATS` to keep it consistent with the actual implementation (as it was before)

Reply:
I guess at some point the docs will need to be updated... not sure. 🤔 

Out of scope for this PR definitely.

Reply:
I agree that code in docs need to be updated to reflect the actual code, but maybe in a separate PR and using something like  `blacken-docs`.

Reply:
Sorry, meant to mention that I also considered this out of scope for this PR. Was just something that came to mind on seeing this.